### PR TITLE
Add initial notes on security

### PR DESCRIPTION
**Pull request by bobturneruk**
These will be useful for developers and infrastructure engineers to refer to. Resolves https://airqoteam.atlassian.net/browse/PLAT-429 (at a high level) - see Jira for more details.

### CODE REVIEW

**Review by NoahNsimbe — APPROVED** [review]
LGTM

File: security.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+“Least Privilege Access” should be employed e.g. if a microservice only needs to read a subset of data, it should do so with a read only account that can only access the subset.
+
+## Encryption
+
+Data should be encrypted in-flight, encryption at-rest should be considered for personally identifiable or otherwise private data in line with local and international law.
+
+## Anti-virus
+
+The plaform largely runs on Linux, so anti virus is a mainly concern for clients. It must be ensured that the plaform does not pass malicious code to clients.
```
Comment (ja-odur) [inline security.md:21]: There is a slight typo here. otherwise everything looks good.

**Review by ja-odur — APPROVED** [review]
LGTM

File: security.md (hunk 1 above)
Comment (bobturneruk) [reply security.md]: Thanks @ja-odur!

**Review by lilly-muyama — APPROVED** [review]
LGTM

**Comment by bobturneruk** [comment]
Thanks all. With 3 approvals now I will rebase and merge.

**Pull request merged**